eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/ResourceCollectionClientProvider.cs

```diff
@@ -109,6 +110,8 @@ private static void InitializeMethods(
 
         protected override string BuildName() => $"{ResourceName}Collection";
 
+        protected override FormattableString BuildDescription() => $"A class representing a collection of <see cref=\"{_resource.Type}\"/> and their operations. Each <see cref=\"{_resource.Type}\"/> in the collection will belong to the same instance of <see cref=\"{GetParentResourceTypeName()}\"/>. To get a {Name} instance call the Get{ResourceName.Pluralize()} method from an instance of <see cref=\"{GetParentResourceTypeName()}\"/>.";
```

Comment:
similar comment here, please use the format specifier for cref and change the contain related to parent resource with TODO